lettuce/flows/taylorgreen.py

```diff
@@ -37,6 +37,12 @@ def grid(self):
     def boundaries(self):
         return []
 
+    @property
+    def res(self):
```

Comment:
1. I don't really think we need this
2. prefer longer but more understandable names (resolution instead of res)
I know I have some things like rho, p, u, j, tau, f, MRT, BGK in the code. However, these all refer to well-established terms and mathematical symbols in the LBM literature. It was a conscious decision to not write these names out because  IMHO it makes the code more readable. For all non-standard names, don't use abbreviations if possible.